docs/source/usage/projections.rst

```diff
@@ -135,27 +135,28 @@ Note that ``+over`` does **not** disable ``+lon_wrap``.
 Prime Meridian
 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-A prime meridian may be declared indicating the offset between the prime
-meridian of the declared coordinate system and that of greenwich.  A prime
-meridian is declared using the "pm" parameter, and may be assigned a symbolic
-name, or the longitude of the alternative prime meridian relative to greenwich.
+A prime meridian may be declared indicating the longitude offset between
+the prime meridian of the declared coordinate system and that of greenwich.
+A prime meridian is declared using the ``+pm`` parameter, and may be assigned
+an angle in DMS or decimal degrees format, or as a hard-coded name.
```

Comment:
I think an example of not using a hard-coded name would be useful. Something like

```
echo 13.5 45.0 | proj +proj=merc +pm=13.5         
0.00    5591295.92
```
could be placed in the example section below.

Reply:
Done, but modified slightly. Further mods welcome.